<?xml version="1.0"?>
<launch>
  <arg name="output" default="screen"/>
  <arg name="respawn" default="false"/>
  <arg name="start_manager" default="false"/>
  <arg name="start_server" default="false"/>
  <arg name="nodelet_manager" default=""/>
  <arg name="launch-prefix" default=""/>

  <arg name="default_inputs" default="merged_stable"/>

  <node name="uwds_server"
        pkg="nodelet" type="nodelet"
        args="load uwds/UwdsServerNodelet $(arg nodelet_manager)"
        respawn="$(arg respawn)"
        output="$(arg output)"
        launch-prefix="$(arg launch-prefix)"
        if="$(arg start_server)"/>

  <node name="overlaytext_situation_publisher"
        pkg="uwds_basic_clients" type="overlaytext_situation_publisher.py"
        respawn="$(arg respawn)"
        output="$(arg output)"
        launch-prefix="$(arg launch-prefix)">
    <rosparam subst_value="true">
      default_inputs: $(arg default_inputs)
    </rosparam>
  </node>
</launch>
